test/fixtures/Shell/main.env

```diff
@@ -0,0 +1,2 @@
+# comment
+export KEY=${ENV}
```

Comment:
Would it be possible to find sample files from "real" applications here instead of using fixture files? Fixture files are not used to train the Bayesian classifier, so I'd really prefer to avoid them as much as possible.

Reply:
@pchaigno Ok, I'll add some!